datamap/mv/core/pom.xml

```diff
@@ -22,7 +22,7 @@
   <parent>
     <groupId>org.apache.carbondata</groupId>
     <artifactId>carbondata-parent</artifactId>
-    <version>1.5.0-SNAPSHOT</version>
+    <version>1.5.1</version>
```

Comment:
please use 1.5.2-SNAPSHOT, not 1.5.1, the same as other module.

Reply:
The package version I downloaded was 1.5.1.
apache-carbondata-1.5.1-rc1.tar.gz

Reply:
The package version you downloaded is from a tag, not from branch-1.5.
the version of all modules on branch-1.5 is 1.5.2-SNAPSHOT, when to be released, version manager will change version to 1.5.1 to tag a version package.

Reply:
@ravipesala @sraghunandan Have you tested MV module before/when we release 1.5.1-rc1? Why didn't change it?